src/app/payment/creditCard/CreditCardCustomField.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { memo, useCallback, useMemo, FunctionComponent } from 'react';
+
+import { TranslatedString } from '../../locale';
+import { FormField, TextInput } from '../../ui/form';
+
+export interface CreditCardCustomFieldProps {
+    additionalClassName: string;
+    autoComplete: string;
+    labelId: string;
+    name: string;
+}
+
+const CreditCardCustomField: FunctionComponent<CreditCardCustomFieldProps> = props => {
```

Comment:
Is a document a credit card? It seems to be quite a generic component by looking at its props.  Maybe `TextField` ?